### PR TITLE
[confighttp] Add `NewDefaultCORSConfig` function

### PR DESCRIPTION
**Pull request by atoulme**
#### Description
Add `NewDefaultCORSConfig` function to initialize the default `confighttp.CORSConfig`

#### Link to tracking issue
#9655

### CODE REVIEW

**Comment by atoulme** [comment]
@andrzej-stencel please review?

File: config/confighttp/confighttp.go [hunk 1]
```diff
@@ -476,6 +476,11 @@ type CORSConfig struct {
 	MaxAge int `mapstructure:"max_age"`
 }
 
+// NewDefaultCORSConfig creates a default cross-origin resource sharing (CORS) configuration.
+func NewDefaultCORSConfig() *CORSConfig {
```
Comment (TylerHelmuth) [inline config/confighttp/confighttp.go:480]: We should use this in `NewDefaultServerConfig`

Comment (atoulme) [reply config/confighttp/confighttp.go:480]: If you merge this, I promise to open a PR to do that

Comment (TylerHelmuth) [reply config/confighttp/confighttp.go:480]: `NewDefaultServerConfig` already exists in `confighttp` right? Can we add it now?

Comment (atoulme) [reply config/confighttp/confighttp.go:517]: yes, done.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**